wiki/Help_Centre/en.md

```diff
@@ -110,7 +111,7 @@ If that player does have any plays across all game modes, a [non-linear function
 
 **Yes, but under certain circumstances.**
 
-We only grant free reverts for a user's direct previous username (the one shown by hovering their name on their profile). Changes to any other previous names must be [purchased via the store](https://osu.ppy.sh/store/products/32). Simply contact accounts@ppy.sh and request your namechange reverted.
+We only grant free reverts for a user's direct previous username (the one shown by hovering their name on their profile). Changes to any other previous names must be [purchased via the store](https://osu.ppy.sh/store/products/32). Simply contact [accounts@ppy.sh](mailto:accounts@ppy.sh) and request your namechange reverted.
```

Comment:
```suggestion
We only grant free reverts for a user's direct previous username. Changes to any other previous names must be [purchased via the store](https://osu.ppy.sh/store/products/32). Simply contact [accounts@ppy.sh](mailto:accounts@ppy.sh) and request your namechange reverted.
``` 
Might still be good to clarify in some way what "direct previous username" means, but the hover thing no longer applies on the new site as it shows all previous usernames.